tests/hpc/slurm_master.pm

```diff
@@ -232,10 +232,9 @@ sub t01_accounting() {
     );
 
     ##Add users TODO: surely this should be abstracted
```

Comment:
You left here that ugly formatted TODO.

Reply:
I need input from seba for that. its his comment and i am not sure what he had in mind.